lib/main.js

```diff
@@ -1,37 +1,38 @@
-"use strict";
+'use strict'
 
-var fs = require("fs");
+var fs = require('fs')
```

Comment:
[`semi`](http://eslint.org/docs/rules/semi.html) would be a good rule to add to `.eslintrc`